### DISCUSSION TITLE
Video Download failed & wifi stops

### DISCUSSION THREAD

**Discussion by Arrea10** [aclap-dev/video-downloadhelper#711] [Help and Q&A]
### Your question

I'm trying to download videos from the Kajabi platform. Usually, I'm able to download 3 or 4 first thing when I open my computer, but after that the downloads start failing and many times (not always) along with the wifi in the computer. What's going on?

### Addon & CoApp version and details

{
  "version": "9.0.1.22",
  "target": "google",
  "channel": "stable",
  "page_url": "chrome://newtab/",
  "error": {
    "details": "SideDownload error: : [tls @ 0x7fce41111ac0] error:0A000119:SSL routines::decryption failed or bad record mac\n[tls @ 0x7fce41111ac0] Unknown error\n[aac_adtstoasc @ 0x7fce41246480] Error parsing ADTS frame header!\n[mp4 @ 0x7fce4051c5c0] Error applying bitstream filters to an output packet for stream #1: Invalid data found when processing input\n[aost#0:1/copy @ 0x7fce4102af80] Error submitting a packet to the muxer: Invalid data found when processing input\n[out#0/mp4 @ 0x7fce4051c500] Error muxing a packet\n",
    "downloadable_id": "downloadable_5544550025572366",
    "error": "coapp_failure",
    "id": "coapp_failure_c3374483-ac22-481d-858c-2779e0669169"
  }
}

**Comment by GrampaWildWilly** [comment]
You need to provide URLs of specific videos exhibiting the problem behavior.  Don't name a web site.  Provide specific URLs of web pages on which your problematic videos reside.

**Reply by Arrea10** [reply]
Thanks. But I think I have seemed to work around the problem: I copy the specific url of each video, open a new page with it, change the end of the address to mp4, and that way it lets me download it! I wish there was a faster way so I could do it without having to go through that many steps! Here's a link p.e.: https://embed-cloudfront.wistia.com/deliveries/49db58aa191b7de3cb017a2638b39d73ef69e860.m3u8
